distribution/test/itests/README.md

```diff
@@ -16,6 +16,13 @@ In short:
 * Use remote debugging to debug tests themselves or anything related to production code
 * Use local debugging to debug the code that sets up and configures the testing environment
 
+## Notes on Solr
+Solr is now runs in a completely separate (3rd) JVM as its own process. The default port is hardcoded to `9994`. (`http://localhost:9994/solr` to access solr UI) Also the actual solr instance lives inside of the `test-itest-ddf/target/solr` folder and **not** in the exam folder. Thus since there is not a per exam folder instance of solr we do not currently support running multiple itests at once. 
```

Comment:
MS Word says `completely` is unnecessary here. 
Also Thus should have a `,` after it.

Reply:
`Completely` gone.
`Thus` removed entirely.